assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -13,9 +13,11 @@ import ContribAmounts from 'components/contribAmounts/contribAmounts';
 import PayPalContributionButton from 'containerisableComponents/payPalContributionButton/payPalContributionButton';
 import TermsPrivacy from 'components/legal/termsPrivacy/termsPrivacy';
 
+import { contribCamelCase } from 'helpers/contributions';
```

Comment:
What is the difference between `getContribKey` and `contribCamelCase`?  Is it posisble to rename `contribCamelCase` to `getContribKey` and delete `getContribKey` from this file?

Reply:
Looks like they're literally the same function, apparently we *really* like converting things to camelCase. Ok, I've deleted `getContribKey`, good catch.

Reply:
e814a19